<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>pqSampleScalarWidget</class>
 <widget class="QWidget" name="pqSampleScalarWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>234</width>
    <height>461</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>8</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Add Range</string>
     </property>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>8</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QLineEdit" name="RangeMin" >
          <property name="toolTip" >
           <string>Minimum Value</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="RangeToLabel" >
          <property name="text" >
           <string>to</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="RangeMax" >
          <property name="toolTip" >
           <string>Maximum Value</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="NumValsLabel" >
          <property name="text" >
           <string>Number of Values:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="RangeCount" >
          <property name="sizePolicy" >
           <sizepolicy>
            <hsizetype>7</hsizetype>
            <vsizetype>0</vsizetype>
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip" >
           <string>Number of values to sample within the range</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="AddRange" >
        <property name="toolTip" >
         <string>Add a range of values to the current value list </string>
        </property>
        <property name="text" >
         <string>Add</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2" >
     <property name="title" >
      <string>Add Value</string>
     </property>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>8</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QLineEdit" name="Value" >
          <property name="toolTip" >
           <string>Value to add</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="AddValue" >
          <property name="toolTip" >
           <string>Add a single value to the current set</string>
          </property>
          <property name="text" >
           <string>Add</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3" >
     <property name="title" >
      <string>Values</string>
     </property>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>8</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QListWidget" name="Values" />
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QPushButton" name="DeleteAll" >
          <property name="enabled" >
           <bool>true</bool>
          </property>
          <property name="toolTip" >
           <string>Remove all values from the current set</string>
          </property>
          <property name="text" >
           <string>Delete All</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="DeleteSelected" >
          <property name="enabled" >
           <bool>true</bool>
          </property>
          <property name="toolTip" >
           <string>Delete selected values from the current set</string>
          </property>
          <property name="text" >
           <string>Delete Selected</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
     <widget class="QWidget" name="widget" >
      <property name="geometry" >
       <rect>
        <x>11</x>
        <y>25</y>
        <width>207</width>
        <height>229</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <resources/>
 <connections/>
</ui>
